app/components/point-input.jsx

```diff
@@ -0,0 +1,191 @@
+    const [isDescFocused, setIsDescFocused] = useState(false)
+    const textareaRef = useRef(null);
+
+    useEffect(() => {
+        autosize(textareaRef.current)
+
+        return () => {
+            autosize.destroy(textareaRef.current)
+        }
+    }, [])
+
+    function getDescWordCount(inputText) {
+        if (!inputText) return 0
+        return inputText.trim().split(/\s+/).length;
+    }
+
+    function getSubjCharCount(inputText) {
+        return inputText.length
+    }
+
+    const isDescValid = inputText => {
+        const wordCount = getDescWordCount(inputText)
+        return wordCount > 0 && wordCount <= maxWordCount
+    }
+
+    const isSubjValid = inputText => {
+        const charCount = getSubjCharCount(inputText)
+        return charCount > 0 && charCount <= maxCharCount
+    }
+
+    const handleSubjectChange = (value) => {
+        setSubject(value)
+        setSubjCharCount(getSubjCharCount(value))
+    }
+
+    const handleDescriptionChange = (value) => {
+        setDescription(value)
+        setDescWordCount(getDescWordCount(value))
+    }
+
+    const handleDescriptionBlur = () => {
+        setIsDescFocused(false)
+        onDone((isSubjValid(subject) && isDescValid(description)), ({ subject: subject, description: description }))
+    }
+
+    return (
+        <>
+            <div className={classes.container}>
+                <input
+                    type="text"
+                    placeholder='Type some thing here'
+                    value={subject}
+                    onChange={(e) => handleSubjectChange(e.target.value)}
+                    onFocus={() => setIsSubjFocused(true)}
+                    onBlur={() => setIsSubjFocused(false)}
+                    className={subjCharCount > maxCharCount ? classes.subject + ' ' + classes.errorInput: classes.subject}>
+
+                </input>
+                {(isSubjFocused || subjCharCount > maxCharCount) && (<span
+                    className={classes.wordCount}
+                >
+                    {subjCharCount} / {maxCharCount}
+                </span>)}
+
+                <textarea
+                    ref={textareaRef}
+                    placeholder="Description"
+                    value={description}
+                    onChange={(e) => handleDescriptionChange(e.target.value)}
+                    onBlur={handleDescriptionBlur}
+                    onFocus={() => setIsDescFocused(true)}
+                    className={descWordCount > maxWordCount ? classes.description + ' ' + classes.errorInput: classes.description}>
+                </textarea>
+                {(isDescFocused || descWordCount > maxWordCount) && (<span
+                    className={classes.wordCount}
+                >{descWordCount} / {maxWordCount}</span>)}
+            </div>
+        </>
+    )
+};
+
+
+const useStyles = createUseStyles(theme => ({
+
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        top: '4.0625rem',
```

Comment:
top and left have no effect in this case, please remove.  And in general it would be up to the parent to position the component within it (what top and left are doing) - which is why we allow the parent to pass className.